etc/org.opencastproject.index.service.impl.util.PublicationEventUpdateHandler.cfg

```diff
@@ -0,0 +1,14 @@
+# A comma separated list of publication channel identifiers to automatically republish to when saving to the asset
+# manager.
+# The available publication channel identifiers depends on your system configuration.
+# Common publication channels are:
+#   - engage-player: Standard publication for players
+#   - interal: Opencast internal publication (mainly for the editor)
+#   - api: for the external api
+# Default value: <empty>
```

Comment:
So by default we don't automatically republish?  This should be made clear here.